metricbeat/helper/server/http/http.go

```diff
@@ -130,6 +148,11 @@ func (h *HttpServer) handleFunc(writer http.ResponseWriter, req *http.Request) {
 
 	case "GET":
 		writer.WriteHeader(http.StatusOK)
-		writer.Write([]byte("HTTP Server accepts data via POST"))
+		if h.server.TLSConfig != nil && h.server.TLSConfig.Certificates != nil {
```

Comment:
```suggestion
		if h.server.TLSConfig != nil {
```

Reply:
This can't be done. I've tested and for some reason, HTTP server create a TLSConfig, after calling ListenAndServe(), so the tests where printing the wrong message if we remove that.

Reply:
Oh, OK. Then I'm glad you tested it!